lib/eslint-rails/text_formatter.rb

```diff
@@ -5,7 +5,7 @@ module ESLintRails
   class TextFormatter
 
     WARNING_TEMPLATE = <<-TEXT.strip_heredoc
-      <%= line %>,<%= column %> <%= severity %>! [<%= rule_id %>] <%= message %>
+      <%= line %>,<%= column %> <%= severity %>! [<%= rule_id %>] <%= message %> | <%= link %>
     TEXT
```

Comment:
haha I don't think we use `WARNING_TEMPLATE`. I think that's left over from a prototype. We should confirm and update/remove/clean up that stuff.